backend/applications/tests/test_applications_api.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from rest_framework.reverse import reverse
+
+from applications.api.v1.serializers import ApplicationSerializer
+from applications.enums import ApplicationStatus
+from applications.tests.factories import ApplicationFactory
+
+
+@pytest.mark.django_db
+def test_applications_list(api_client):
+    ApplicationFactory()
```

Comment:
Just as a suggestion for the future. If you wanted to "clean up" this a tiny bit, then something like `pytest-factoryboy` could remove this one line and turn the function to `def test_applications_list(api_client, application):` that way you could use fixtures only in the function parameters instead of calling the factory in the test.

Reply:
Good point!